packages/shared/src/components/cards/squad/SquadList.tsx

```diff
@@ -62,7 +62,7 @@ export const SquadList = ({
         copy={{ join: 'Join', view: 'View' }}
         data-testid="squad-action"
         showViewSquad
-        buttonVariants={[ButtonVariant.Secondary, ButtonVariant.Secondary]}
+        buttonVariants={[ButtonVariant.Secondary, ButtonVariant.Float]}
```

Comment:
View buttons should be Float.

![image](https://github.com/user-attachments/assets/99f3cf8e-c850-4238-86d4-92a298df20fc)